docs/content/querying/topnquery.md

```diff
@@ -48,7 +48,7 @@ A topN query object looks like:
   "postAggregations": [
     {
       "type": "arithmetic",
-      "name": "sample_divide",
+      "name": "average",
```

Comment:
I think it makes sense, `sum(some_metrics) / sum(count)` is an average for a rolled up datasource.